src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -60,7 +72,7 @@ const SearchResult = ({ form }) => {
 
       <dd className="vads-u-padding-bottom--3">
         <a
-          download={form.attributes.url}
+          download={deriveDownloadName(form)}
```

Comment:
Actually, I think we should just use `form.id`, or nothing (just a plain `download` attribute) instead of deriving the name. I just realized that some of the form titles are massive lol.

Reply:
haha all good, that's what i had thought too 🙂